index.js

```diff
@@ -0,0 +1,57 @@
+import BookClass from './modules/javascript/book.js';
+import showSection from './modules/javascript/nav.js';
+import selectors from './modules/javascript/selectors.js';
+import { DateTime } from './modules/javascript/luxon.min.js';
+
+// Lets get the add book button
+const addButton = document.querySelector('#add-button');
+// Lets add a click event listener with which we will perform the add logic for the new book
+addButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  const form = document.getElementById('add-book-form');
+  if (!form.reportValidity()) {
+    return;
+  }
+  // lets get the new title
+  const newTitle = document.querySelector('#book-title-input').value;
+  // now lets get the new author
+  const newAuthor = document.querySelector('#book-author-input').value;
+  // lets add it to the books object at this point
+  const book = new BookClass();
+  book.addBook(newTitle, newAuthor);
+  // okay great. Lets reset the form now
+  form.reset();
+});
+
+/* eslint-disable */
+function removeBook(key) {
+  const book = new BookClass();
+  book.removeBook(key);
+}
```

Comment:
- [ ] According to the project requirement, Kindly change this function to an arrow function